### PR TITLE
Feature/bulk data message

### PR DESCRIPTION
**Pull request by noahmanger**
This addresses a couple interrelated issues that have surfaced over time:
- Users have expressed difficulty figuring out how to control the dates and enter contributor names
- The link in to the bulk data from the disabled export button tooltip is actually unclickable and not obvious

This implements the message pattern as designed in https://github.com/18F/openFEC-web-app/issues/2074 and also pulls out the contributor name and date filters to the top, open panel:
image

It makes a similar change to disbursements:
image

The pattern we're going for is to always expose filters for the two parties of any given transaction in the top, open panel (we already do this with independent expenditures) as well as any constraining factors, like time period. 

Other details:
- The message links to https://www.fec.gov/data/advanced/?tab=other for bulk downloads
- The "Committee name or ID" field was relabelled to "Recipient name or ID" on receipts and individual contributions
- The "Committee name or ID" field was relabelled to "Spender name or ID" on disbursements (same as it is on independent expenditures
- The "Time period" filter was relabelled to "Transaction time period" to make clearer that it's referring to the time period the transaction was made within

cc @nickykrause

### CODE REVIEW

**Comment by noahmanger** [comment]
Thanks @jenniferthibault for pairing with me to figure this out in code, and @nickykrause for the smart design ideas.

**Comment by jenniferthibault** [comment]
One small addition: we also relabelled the accordions from `Transaction information` to `Transaction details` to re-emphasize the model that there are some basic filters at the top, but more _detailed_ options in each category below. 

@nickykrause still very open to your thoughts and reactions—just because it's in a PR doesn't mean it's decided, it was just much, much less tedious to move the components in code than rebuilding them in the design files.

**Comment by noahmanger** [comment]
And this is how it looks on candidate / committee pages:
image

**Comment by nickykrause** [comment]
@jenniferthibault @noahmanger all looks good to me, but I appreciate the tag! 👍

File: static/js/modules/tables.js [hunk 1]
```diff
@@ -30,10 +30,10 @@ var downloadCapFormatted = helpers.formatNumber(DOWNLOAD_CAP);
 var MAX_DOWNLOADS = 5;
 var DOWNLOAD_MESSAGES = {
   recordCap:
-    'Exports are limited to ' +
+    'Use <a href="' + window.BASE_PATH + '/advanced?tab=other">' +
```
Comment (ccostino) [inline static/js/modules/tables.js]: What is `BASE_PATH` defined as here?

Comment (noahmanger) [inline static/js/modules/tables.js]: On non-local envs: `/data`

**Pull request merged**